src/main/java/com/langserver/devtools/intellij/liberty/lsp/BoostrapPropertiesFileType.java

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
```

Comment:
You probably meant to call this `BootstrapPropertiesFileType`. Similarly for `BootstrapPropertiesLanguage`.

Reply:
Yes thank you, fixed